src/templates/contentScript.ts

```diff
@@ -0,0 +1,58 @@
+import { Template } from "./index";
+
+export const ContentScript: Template = {
+  id: "contentScript",
+  name: "Content Script",
+  manifest: {
+    name: "Content Script",
+    description: "Basic Content Script Extension",
+    backgroundScripts: ["background.js"],
+    contentScripts: [
+      {
+        matches: ["<all_urls>"],
+        js: ["content_script.js"],
+      },
+    ],
+  },
+  files: [
```

Comment:
Do you mind merging main? I just fixed a bug for templates with multiple TS files: https://github.com/oliverdunk/web-extension-playground/pull/10.